src/solids4FoamModels/fluidModels/pimpleFluid/pimpleFluid.esi.C

```diff
@@ -192,6 +193,9 @@ bool pimpleFluid::evolve()
                 {
                     #include "meshCourantNo.H"
                 }
+
+                // Update flux in FSI interface with Robin BC
+                robin_.setInterfaceFluxToZero(phi);
```

Comment:
In this portion, you started to modify the ESI but is not complete yet right? I mean, the other functions of robinInterfaceHelper are still missing from the code.